charts/hlf-k8s/templates/rbac.yaml

```diff
@@ -42,9 +42,7 @@ rules:
     - watch
     - list
     - create
-    {{- if .Values.hooks.deleteSecrets.enabled }}
```

Comment:
Because there is a job hook to remove chaincode ccid secrets now :)

Reply:
Shouldn't this condition be kept but modified instead (so that if any of the hooks is enabled we add the "delete secrets" permission).

My understanding is that now the "delete" permission is granted even when it's not needed (i.e. when both hooks are disabled)